Get-LockboxAudits.ps1

```diff
@@ -30,15 +30,23 @@
     (Optional) Include this flag to group audit events by the audit Action type. Cannot be combined with other groupBy parameters.
 
 .PARAMETER getAdminCaseAudits
-    (Optional) Include this flag to show audits that took place in the Admin Case configuration workspace. This will show audits for system configuration changes such as Login events, User and Group object changes, and Instance Setting edits.
+    (Optional) Include this flag to show audits that took place in the Admin Case configuration workspace. 
+    This will show audits for system configuration changes such as Login events, User and Group object creations or changes, and Instance Setting modifications.
+    NOTE: Including this flag will greatly increase the number of audit entries in the results.
 
 NOTES:
     - The restUserName and restPassword need to be for an account which has both Audit API access, such as System Administrator group members, and a Password Login Method.
     - To capture all events up to the script run time, it is recommended that the upperRangeDate parameter be set to tomorrow's date.
+    - Both the $lowerRangeDate and $upperRangeDate are in UTC time.
+    - The required script execution time is in proportion to the number of Relativity Employee user accounts in the R1 environment and the number of audits found. 
     - Disclaimer for RelativityOne environment usage:
         - This script is limited to public Audit APIs for retrieving audit events.
-        - Due to technical limitations of the public Audit APIs and the automatic deletion of Relativity Employee accounts, audits of deleted Relativity Employees are only available via this script for as long as the employee accounts exist in the Relativity environment.
-        - It is recommended that the script be run at least daily to capture Relativity Employee audits before the user accounts are removed. The script can be run as often as desired to ensure all Relativity Employee access is captured.
+        - Due to technical limitations of the public Audit APIs and the automatic deletion of Relativity Employee accounts, audits of Relativity Employees are only 
+          available via this script for as long as the employee accounts exist in the Relativity environment. Once a Relativity Employee account has been deleted, this script

```

Comment:
sounds we should talk to cloud sec and whoever owns the user deletion code to prevent a relativity user being deleted from tenant environment. they should be only disabled instead.

Reply:
I don't think this needs to be a change to user-deletion logic - this would be part of CloudSec fixing the problem of "why do we re-create the user every time break-glass happens".

On the other hand, we don't want to clutter up the environment for tenants - even disabled users will be visible when they try to do their own user-management.

Reply:
we are talking about being transparent to customers. I feel keeping a relativity user will be part of it.
for cloud sec, what can we do without a deleting a user after break-glass expires, other than disable it to make sure the user  have the same artifact id if he needs the access again?
for now, we can't promise it is 100% transparent if a user is deleted.

Reply:
Sure - but I believe CloudSec already has a future roadmap item for "this process needs to be better". Our team will benefit from it, but we aren't solving it.

Reply:
Nah, I am not trying to solve it. I am just trying to give some feedback(complaints) to the api/db I am using as a customer.

Then it will come back as a customer, I need to work around the issue than kicking the can down to the owner. meh....

Reply:
I agree this is an issue that should be looked at more. The limitations will remain for the mean time.

Reply:
just a test from vs code...